rdagent/scenarios/data_science/dev/runner/eval.py

```diff
@@ -54,6 +54,9 @@ def evaluate(
 
         # execute workflow
         stdout, execute_ret_code = implementation.execute_ret_code(env=env, entry="python -m coverage run main.py")
+        running_time = re.search(r"Total running time: (\d+\.\d+) seconds.", stdout).group(1)
+        implementation.inject_files(**{"running_time.txt": running_time})
```

Comment:
If we have already attrribute in feedback, we don't need a extra file in the worksapce